src/server/server.js

```diff
@@ -107,6 +107,51 @@ const createStock = quote => {
   };
 };
 
+const calculateDateYearsInPast = years => {
+  const year = new Date().getFullYear() - years;
+  const date = new Date();
+  date.setFullYear(year);
+  return date;
+};
+
+const padSingleDigitWithZero = value => {
```

Comment:
logic is a little off here I think.  What if I passed in value = 'hailey'?  No error would be thrown because it's a string but the `parseInt` would fail.  `parseInt` should be done first and then check its result and throw an error if NaN.